src/api/trackerTrackedEntities.ts

```diff
@@ -131,13 +154,33 @@ export type TrackedEntitiesParamsBase = {
     includeDeleted: boolean;
     includeAllAttributes: boolean;
     potentialDuplicate: boolean;
+    order: TrackedOrderBase[];
 };
 
+export type TrackedOrderBase = {
+    direction: "asc" | "desc";
+} & (TrackedOrderFields | TrackedAttributesFields);
+
+export type TrackedOrderFields = {
+    type: "field";
+    field:
+        | "createdAtClient"
+        | "createdAt"
+        | "enrolledAt"
+        | "inactive"
+        | "trackedEntity"
+        | "updatedAt";
+};
+
+export type TrackedAttributesFields = { type: "id"; id: Id };
```

Comment:
The name of type makes it clear within the module, but the plain object is created (example: `{ type: "id", id: "wMhqqPLb7pP", direction: "desc" }`), we don't what's the model of the id. Maybe `type: "trackedEntityAttributeId"`?